streams/src/test/java/org/apache/kafka/streams/integration/WordCountIntegrationTest.java

```diff
@@ -65,9 +67,9 @@ public void shouldCountWords() throws Exception {
         List<String> inputValues = Arrays.asList("hello", "world", "world", "hello world");
         List<KeyValue<String, Long>> expectedWordCounts = Arrays.asList(
             new KeyValue<>("hello", 1L),
+            new KeyValue<>("hello", 2L),
```

Comment:
This goes with the sorting comment below. Don't we want to verify order, i.e., that values received come back exactly in the order we sent them? Hence I recommended avoiding sorting.